packages/boot/test/bootstrapTests/supports.ts

```diff
@@ -215,13 +215,21 @@ export const makeProposalExtractor = ({ childProcess, fs }: Powers) => {
   const getPkgPath = (pkg, fileName = '') =>
     new URL(`../../../${pkg}/${fileName}`, import.meta.url).pathname;
 
-  const runPackageScript = async (pkg, name, env) => {
-    console.warn(pkg, 'running package script:', name);
-    const pkgPath = getPkgPath(pkg);
-    return childProcess.execFileSync('yarn', ['run', name], {
-      cwd: pkgPath,
+  const runPackageScript = async (outputDir, pkg, location, env) => {
```

Comment:
A "package script" is a line in "scripts" in package.json. This PR removes those so this function name is no longer accurate.

The `location` can have the package in it so there's no longer need for a separate `pkg` param.

```suggestion
  const runProposalBuilder = async (outputDir, builderPath, env) => {
```

Then the call in `liquidation.js` would be like,
```js
  const proposal = await buildProposal({
    builderScript: '@agoric/builders/scripts/inter-protocol/add-STARS.js',
  });
```
You can get a pathname for that with something like this: https://github.com/Agoric/agoric-sdk/blob/d4b32f129868ed2faa32f1bcad51b5d2248d4c14/packages/boot/test/upgrading/test-upgrade-contracts.js#L15-L17